app/authorities/qa/authorities/iucat_libraries.rb

```diff
@@ -0,0 +1,33 @@
+module Qa::Authorities
+  class IucatLibraries < Base
+    include WebServiceBase
+
+    def search(id)
+      all.select { |library| library[:code].match(id.to_s) }
+    end
+
+    def find(id)
+      data_for(id)[:library] || {}
+    end
+
+    def all
+      data_for('all')[:libraries] || []
+    end
+
+    private
+      def api_url
+        { development: 'http://iucat-feature-tadas.uits.iu.edu/api/library',
+          test: 'http://iucat-feature-tadas.uits.iu.edu/api/library',
+          production: 'http://iucat.iu.edu/api/library' }.with_indifferent_access[Rails.env]
```

Comment:
Would it be too late before sprint end on 1-15 to parametrize the values for api_url via config/secrets?

Reply:
@randalldfloyd Not at all -- I should have done that step before putting up the PR

Reply:
@randalldfloyd added a commit